session/bootstrap.go

```diff
@@ -238,7 +239,7 @@ const (
 func bootstrap(s Session) {
 	b, err := checkBootstrapped(s)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(err.Error())
```

Comment:
Do we need to add more information to these logs?

Reply:
There're too many of them. It is better to do it in another PR.